ehr2meds/synthetic_data_generation/generate.py

```
@@ -99,6 +93,17 @@ def greater_than_datetime(min_date, end=2020):
     return min_date + timedelta(seconds=random_seconds)
 
 
+# Specialized functions for SP
+def gestationsalder_with_fraction():
+    weeks = random.randint(0, 42)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)